app/bibleview-js/src/lang/default.yaml

```diff
@@ -68,6 +68,19 @@ verseMemorizeLong: Memorize Bible verses
 verseNoteLong: Write a note
 addBookmark: Bookmark
 selectCustomIconTitle: Select custom marker icon
+bookmarkSettingsTitle: Configure bookmark settings
+customIconLabel: Custom icon
+editActionLabel: Edit action
+editActionModeLabel: Mode
+editActionModeNone: None
+editActionModeAppend: Append text
+editActionModePrepend: Prepend text
+editActionModeReplace: Replace text
+editActionContentLabel: Content
+editActionContentPlaceholder: Enter text to be added to the bookmark...
+insertParagraphBreak: Insert paragraph break
+insertSubtitle: Insert subtitle
+formatHelp: "Use <br/> for paragraph breaks and <subtitle>text</subtitle> for subtitles"
```

Comment:
We should insert tags / examples to the translated text, not have them translatable